zipkin-autoconfigure/metrics-prometheus/src/test/java/zipkin/autoconfigure/metrics/PrometheusMetricsAutoConfigurationTest.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright 2015-2016 The OpenZipkin Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package zipkin.autoconfigure.metrics;
+
+import org.junit.Test;
+import org.springframework.boot.actuate.endpoint.PublicMetrics;
+import org.springframework.boot.actuate.metrics.Metric;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import java.util.Collections;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+public class PrometheusMetricsAutoConfigurationTest {
+
+    @Test
+    public void correctHttpResponse() throws Exception {
+        final PublicMetrics publicMetrics = () -> Collections.singleton(new Metric<Number>("mem.free", 1024));
```

Comment:
no big deal, but we don't put final on variables even if it is explicit and there are good reasons why some folks make conventions like that.

leaving them out keeps line shorter and focuses more attention on the test code (at risk of someone doing the shell-game with references)